metricbeat/module/postgresql/statements/statements.go

```diff
@@ -0,0 +1,57 @@
+package statements
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/postgresql"
+
+	// Register postgresql database/sql driver
+	_ "github.com/lib/pq"
+)
+
+// init registers the MetricSet with the central registry.
+// The New method will be called after the setup of the module and before starting to fetch data
+func init() {
+	mb.Registry.MustAddMetricSet("postgresql", "statements", New,
+		mb.WithHostParser(postgresql.ParseURL),
+		mb.DefaultMetricSet(),
```

Comment:
Does it generate events on each executed statement? Maybe this is too much data to be enabled by default.